lib/tasks/lumen.rake

```diff
@@ -387,11 +387,15 @@ namespace :lumen do
     # work descriptions. We only look at works before a certain date (shortly
     # after our deploy date for this feature) because anything after that date
     # will be auto_redacted as part of the save process.
-    works = Work.where('updated_at < (?)', Date.new(2018, 12, 31)).limit(1000)
+    works = Work.where('updated_at < (?)', Date.new(2018, 12, 21))
+                .order(:id)
+                .limit(100_000)
+    updated_works = []
     works.each do |work|
       # Make sure that updated_at changes, so we don't re-redact this work.
       work.touch
       work.save # calls auto_redact
```

Comment:
Won't it call https://github.com/berkmancenter/lumendatabase/blob/15743_work_redaction/app/models/work.rb#L77?

Reply:
Whoa. You're totally right and I forgot I wrote that. Sweet! I'll go and simplify this.